dbtmetabase/__init__.py

```diff
@@ -78,58 +139,103 @@ def main(args: list = None):
     )
     parser.add_argument(
         "--mb_https",
-        metavar="HTTPS",
-        type=bool,
-        default=True,
-        help="use HTTPS to connect to Metabase instead of HTTP",
+        action="store_true",
+        default=False,
+        help="Use HTTPS to connect to Metabase instead of HTTP",
+    )
+    parser.add_argument(
+        "--mb_verify",
+        metavar="CERT",
+        help="Path to certificate bundle used by Metabase client",
+    )
+    parser.add_argument(
+        "--database",
+        metavar="ALIAS",
+        required=True,
+        help="Target database name as set in Metabase (typically aliased)",
+    )
+    parser.add_argument(
+        "--dbt_database",
+        metavar="DB",
+        required=True,
+        help="Target database name as specified in dbt",
     )
     parser.add_argument(
-        "--database", metavar="DB", required=True, help="target database name"
+        "--schema",
+        metavar="SCHEMA",
+        help="Target schema name. Cannot be specified with --schema_excludes.",
     )
     parser.add_argument(
-        "--schema", metavar="SCHEMA", required=True, help="target schema name"
+        "--schema_excludes",
+        help="Target schemas to exclude. Cannot be specified with --schema. Will sync all schemas not excluded.",
     )
     parser.add_argument(
         "--sync",
-        metavar="ENABLE",
-        type=bool,
-        default=True,
-        help="synchronize Metabase database before export",
+        action="store_true",
+        default=False,
+        help="Synchronize Metabase database before export. Useful for CI/CD deployment after dbt run.",
     )
     parser.add_argument(
         "--sync_timeout",
         metavar="SECS",
         type=int,
-        default=30,
-        help="synchronization timeout (in secs)",
+        help="Synchronization timeout (in secs). If set, we will fail hard on synchronization failure; if not set, we will proceed after attempting sync regardless of success.",
     )
     parser.add_argument(
         "--includes",
         metavar="MODELS",
         nargs="*",
         default=[],
-        help="model names to limit processing to",
+        help="Model names to limit processing to",
     )
     parser.add_argument(
         "--excludes",
         metavar="MODELS",
         nargs="*",
         default=[],
-        help="model names to exclude",
+        help="Model names to exclude",
+    )
+    parser.add_argument(
+        "--include_tags",
+        action="store_true",
+        default=False,
+        help="Append tags to Table descriptions in Metabase.",
+    )
+    parser.add_argument(
+        "--docs",
+        metavar="DOCS URL",
+        help="Pass in url to dbt docs site. Appends dbt docs url for each model to Metabase table description.",
+    )
+    parser.add_argument(
+        "--verbose",
+        action="store_true",
+        default=False,
+        help="Verbose output.",
```

Comment:
Not a big deal, but some strings in `help` end with a period (`.`), and others don't. Not sure which one to use, though. Maybe the author of the library has a preferred format?

Reply:
I went ahead and normalized them all to not have a period as they were prior. If lib author has a pref change though we can go with that